geometry_msgs/PoseArray faces
geometry_msgs/PoseArray inCamera
geometry_msgs/PoseArray approaches
float32[] angles